docs/maintainers/issue-management.md

```diff
@@ -9,28 +9,28 @@ We use the following labels for issue classifications:
 
 * `Issue-Bug`: the issue is reporting a bug
 * `Issue-Discussion`: the issue may not have a clear classification yet.
-  The issue may generate a [RFC][ln-rfc] or maybe be reclassified as a bug or enhancement.
+  The issue may generate an [RFC][ln-rfc] or maybe be reclassified as a bug or enhancement.
```

Comment:
The reason for `an` instead of `a` in `an RFC` is that `RFC` is pronounced `arr, eff, cee` and thus `an arr-eff-cee` is correct even though the first letter of `RFC` appears to be a consonant.